### PR TITLE
6392 zdb: Introduce -V for verbatim import

### PR DESCRIPTION
**Pull request by ryao**
Reviewed by: Brian Behlendorf behlendorf1@llnl.gov
Reviewed by: Matthew Ahrens mahrens@delphix.com

When given a pool name via -e, zdb would attempt an import. If it
failed, then it would attempt a verbatim import. This behavior is
not always desirable so a -V switch is added to zdb to control the
behavior. When specified, a verbatim import is done. Otherwise,
the behavior is as it was previously, except no verbatim import
is done on failure.

https://github.com/zfsonlinux/zfs/commit/a5778ea2427bd340e3b4f697d9b6e1452bd71909

Depends on openzfs/openzfs#21.

### CODE REVIEW

File: usr/src/cmd/zdb/zdb.c [hunk 1]
```diff
@@ -3529,15 +3529,26 @@ main(int argc, char **argv)
 	char *target;
 	nvlist_t *policy = NULL;
 	uint64_t max_txg = UINT64_MAX;
+	int flags = ZFS_IMPORT_MISSING_LOG;
 	int rewind = ZPOOL_NEVER_REWIND;
+	char *spa_config_path_env;
+	const char *opts = "bcdhilmMI:suCDRSAFLVXx:evp:t:U:P";
```
Comment (ahrens) [inline usr/src/cmd/zdb/zdb.c]: I would suggest that if you're going to move this, you also go ahead and sort it, like https://github.com/openzfs/openzfs/pull/29 proposes.

**Comment by ahrens** [comment]
Changes look good to me.  @grwilson do you want to take a look?

**Comment by ahrens** [comment]
@zettabot go

**Comment by grwilson** [comment]
LGTM

**Comment by ahrens** [comment]
@ryao  It looks like you will need to update zdb_001_neg (see the test results; even without the full log I think the problem is clear).

**Comment by ahrens** [comment]
Anyone interested in working on this?  Perhaps @pzakha?  Needs to be rebased and to fix zdb_001_neg.  If I don't hear anything, I'll close this on 1/13/2017.

**Comment by yuripv** [comment]
Will take this one as well.

**Comment by pzakha** [comment]
@yuripv Thanks for taking this. @ahrens sorry I missed this email during the holidays

**Comment by ahrens** [comment]
Superseded by https://github.com/openzfs/openzfs/pull/314

**Pull request closed without merging**